sections/semantics-embedded-content.include

```diff
@@ -6122,10 +6122,10 @@ zero or more <{track}> elements, then
   var videoSection = document.getElementById('video');
   var videoElement = document.createElement('video');
   var support = videoElement.canPlayType('video/x-new-fictional-format;codecs="kittens,bunnies"');
-  if (support != "probably" &amp;&amp; "New Fictional Video Plugin" in navigator.plugins) {
+  if (support != "probably") {
```

Comment:
This change breaks the branch below. While it looks like this branch will only trigger if canPlayType() returns "maybe", in reality it kills the elseif branch below for the case "" as this evaluates first - the latter never evaluates, so the case where canPlayType() returns "" it will try to create an embed element.

Question is whether this branch should instead check for "maybe", as that would probably suggest the browser natively might support said content type, meaning it should probably try putting it in the video element instead of tossing it over to a plugin and crossing fingers.